docs/core/config/auth/passdb.md

```diff
@@ -442,6 +445,44 @@ a load spike of everybody getting logged in at exactly the same time.
 
 Do not perform any authentication, just store extra fields if user is found.
 
+##### `check_client_fp`
+
+Match client certificate or public key fingerprint.
+
+Requires configuring [[setting,ssl_peer_certificate_fingerprint_hash]], [[setting,auth_ssl_require_client_cert]],
+and [[setting,ssl_server_request_client_cert]] to require a client cert.
+
+This is intended to replace CA certificates with verifying client certificates using fingerprints, or to enforce
+that particular public key or certificate is being used.
+
+To enable this feature, you need
+
+ * [[setting,auth_ssl_require_client_cert,yes]]
+ * [[setting,ssl_server_request_client_cert,any-cert]] if you don't want to validate cert against certificate authority
```

Comment:
This list is 1) missing ssl_peer_certificate_fingerprint_hash, 2) duplicating the list in the "Requires configuring ..." paragraph. Maybe remove the "Requires configuring ..."?